README.md

```diff
@@ -3,12 +3,42 @@
 `rcpputils` is a C++ API consisting of macros, functions, and data structures intended for use throughout the ROS 2 codebase
 
 This package currently contains:
+* Assertion functions
 * Clang thread safety annotation macros
 * Library discovery
 * String helpers
 * File system helpers
 * Type traits helpers
 
+## Assertion Functions
+The [rcpputils/asserts.hpp](rcpputils/include/rcpputils/asserts.hpp) header provides the helper functions:
```

Comment:
Are any or all of these no-ops in a release build?

Reply:
Only `assert_true` becomes a no-op.